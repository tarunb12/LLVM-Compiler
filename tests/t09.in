// For loop test (sum of four squares)

int main() {
    int n = 743
    int i
    int j
    int k
    int l
    for (i = 0; i * i <= n; i++) {
        for (j = i; j * j <= n; j++) {
            for (k = j; k * k <= n; k++) {
                for (l = k; l <= n; l++) {
                    if (i * i + j * j + k * k + l * l == n) printf("%d^2 + %d^2 + %d^2 + %d^2 = %d\n", i, j, k, l, n)
                }
            }
        }
    }
    return 0
}