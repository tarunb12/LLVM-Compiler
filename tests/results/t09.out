1^2 + 2^2 + 3^2 + 27^2 = 743
1^2 + 6^2 + 9^2 + 25^2 = 743
2^2 + 3^2 + 17^2 + 21^2 = 743
2^2 + 15^2 + 15^2 + 17^2 = 743
3^2 + 3^2 + 7^2 + 26^2 = 743
3^2 + 3^2 + 10^2 + 25^2 = 743
3^2 + 3^2 + 14^2 + 23^2 = 743
3^2 + 5^2 + 15^2 + 22^2 = 743
3^2 + 6^2 + 13^2 + 23^2 = 743
3^2 + 7^2 + 18^2 + 19^2 = 743
3^2 + 9^2 + 13^2 + 22^2 = 743
3^2 + 11^2 + 17^2 + 18^2 = 743
5^2 + 9^2 + 14^2 + 21^2 = 743
5^2 + 13^2 + 15^2 + 18^2 = 743
6^2 + 11^2 + 15^2 + 19^2 = 743
7^2 + 9^2 + 17^2 + 18^2 = 743
9^2 + 10^2 + 11^2 + 21^2 = 743
9^2 + 13^2 + 13^2 + 18^2 = 743
